prisma/schema.prisma

```diff
@@ -16,12 +16,27 @@ model User {
   userName String         @unique
   password String
   score    Int
+  role     Role           @default(USER)
   games    UsersOnGames[]
 }
 
+model Admin {
+  id       Int    @id @default(autoincrement())
+  userName String @unique
+  password String
+  role     Role   @default(ADMIN)
+}
```

Comment:
It's not necessary to have an Admin model because you have the `role` property in the User model.